src/connectors/panoptic/panoptic.controllers.ts

```diff
@@ -739,67 +848,79 @@ export async function deposit(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: DepositRequest
-): Promise<string> {
+): Promise<DepositResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
   const asset = await panopticish.deposit(
     wallet,
     req.collateralTracker,
     BigNumber.from(req.assets)
   );
-  return asset;
+  return {
+    shares: asset.shares
+  }; 
 }
 
 export async function getAsset(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: GetAssetRequest
-): Promise<string> {
+): Promise<GetAssetResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
   const asset = await panopticish.getAsset(
     wallet,
     req.collateralTracker
   );
-  return asset;
+  return {
+    assetTokenAddress: asset
+  }; 
 }
 
 export async function getPoolData(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: GetPoolDataRequest
-): Promise<string> {
+): Promise<GetPoolDataResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
-  const shares = await panopticish.getPoolData(
+  const result = await panopticish.getPoolData(
     wallet,
     req.collateralTracker
   );
-  return shares;
+  return {
+    poolAssets: result.poolAssets,
+    insideAMM: result.insideAMM, 
+    currentPoolUtilization: result.currentPoolUtilization
+  };
 }
 
 export async function maxWithdraw(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: MaxWithdrawRequest
-): Promise<string> {
+): Promise<MaxWithdrawResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
-  const limit = await panopticish.maxWithdraw(
+  const result = await panopticish.maxWithdraw(
     wallet,
     req.collateralTracker
   );
-  return limit;
+  return {
+    maxAssets: result.maxAssets
+  };
 }
 
 export async function withdraw(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: WithdrawRequest
-): Promise<string> {
+): Promise<WithdrawResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
-  const shares = await panopticish.withdraw(
+  const response = await panopticish.withdraw(
     wallet,
     req.collateralTracker,
     BigNumber.from(req.assets)
   );
-  return shares;
+  return {
+    shares: response.shares
```

Comment:
Let's rename this to `sharesWithdrawn`

Reply:
Done: b7baad82af6d7f82bbb1a5871e9f8671519172b7